JitsiMeetJS.ts

```diff
@@ -194,6 +196,24 @@ export default {
         Logger.setLogLevel(level);
     },
 
+    /**
+     * Expose rtcstats to the public API. 
+     */
+    rtcstats: {
+      /**
+       * Sends a stats entry to rtcstats server.
+       * @param {string} statsType - The type of stats to send.
+       * @param {Object} data - The stats data to send.
+       */
+      sendStatsEntry(statsType, data) {       
+        RTCStats.sendStatsEntry(statsType, null, data);
+      },
+      /**
+       * Events generated by rtcstats, such as PeerConnections state, and websocket connection state.
+       */
+      events: RTCStats.events
```

Comment:
All event types are defined here: https://github.com/jitsi/lib-jitsi-meet/blob/77e6803f1915174ba4bce2cb36f942cce55d6005/JitsiMeetJS.ts#L112

So let's have those there.

Also, they should be constants, not an event emitter.

I'd say a sample usage should look like this:

```
JitsiMeetJS.rtcstats.on(JitsiMeetJS.events.rtcstats.RTC_STATS_PC_EVENT, ev => ...);
```

Reply:
I've changed event propagation a bit, to match other examples in ljm, let me know if that approach is ok.